### DISCUSSION TITLE
Is this the right place to ask questions about Riverpod usage?

### DISCUSSION THREAD

**Discussion by wujek-srujek** [rrousselGit/riverpod#565] [Q&A] [answered]
I decided to use `Riverpod` for my next app, and have some questions about its usage, observations and feedback (but mostly questions ;d). Is this the right place to ask question like this one https://github.com/rrousselGit/river_pod/discussions/564?

**Comment by julienlebren** [comment, marked as answer]
Yes it is 😉
